ratom_api/settings/local.example.py

```diff
@@ -1,3 +0,0 @@
-from ratom_api.settings.dev import *   # noqa
```

Comment:
Why was this file deleted?

Reply:
Not sure. I might have been trying things out. I'll put it back.